### PR TITLE
Make Bincode's Serializer and Deserializer impls work for any bincode::Options.

### PR DESCRIPTION
**Pull request by tikue**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -369,9 +369,10 @@ pub mod formats {
         #[cfg_attr(docsrs, doc(cfg(feature = "bincode")))]
         pub type SymmetricalBincode<T, O = bincode_crate::DefaultOptions> = Bincode<T, T, O>;
 
-        impl<Item, SinkItem> Deserializer<Item> for Bincode<Item, SinkItem>
+        impl<Item, SinkItem, O> Deserializer<Item> for Bincode<Item, SinkItem, O>
         where
             for<'a> Item: Deserialize<'a>,
+            O: Options + Copy,
```
Comment (vorot93) [inline src/lib.rs:375]: Is `Copy` bound necessary?

Comment (tikue) [reply src/lib.rs:375]: I think Clone is sufficient. Would that be ok? The serialize and deserialize methods take self by value.

Comment (vorot93) [reply src/lib.rs:375]: > The serialize and deserialize methods take self by value.

Ah, this is rather unfortunate. `Clone` would *probably* be better, yes.

Comment (tikue) [reply src/lib.rs]: Done.

**Pull request merged**